packages/api-explorer/src/components/DocReferences/DocReferences.tsx

```diff
@@ -54,8 +55,8 @@ export const DocReferences: FC<DocReferencesProps> = ({
   if (seeTypes.length === 0 && seeMethods.length === 0) return <></>
 
   return (
-    <CollapserCard heading="References">
-      <>
+    <Box mb="xlarge">
+      <CollapserCard heading="References">
```

Comment:
`CollapserCard` expects `children` to be a `ReactElement` so we still need the fragment around the elements here.